edxp-core/src/main/cpp/main/src/config_manager.h

```diff
@@ -68,6 +68,10 @@ namespace edxp {
             return misc_path_ / "framework" / suffix;
         }
 
+        inline static auto GetCachePath(const std::string &suffix = {}) {
+            return misc_path_ / "cache" / suffix;
```

Comment:
does `misc_path_` includes `edxp_xxx` prefix?

Reply:
Also I think it's better to split cache for YAHFA and SandHook

Reply:
> does `misc_path_` includes `edxp_xxx` prefix?

Ofcourse. See how it loads.

Reply:
> Also I think it's better to split cache for YAHFA and SandHook

They have different prefixes.